src/blog/2020-05-15/azure-pipelines-integration.pug

```diff
@@ -0,0 +1,158 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+        When using continuous integration it can be hard to keep track of your test quality. By integrating Stryker in your build pipelines you can ensure your test quality stays at the same level. No matter how many changes your code gets to see.
```

Comment:
```suggestion
        When using continuous integration it can be hard to keep track of quality. By integrating Stryker in your build pipelines you can ensure a continued focus on test quality, no matter how many changes your code gets to see.
```